test/upload-api.test.js

```diff
@@ -33,8 +43,12 @@ test('GET /version', async t => {
 })
 
 test('POST / client can upload a file and list it', async t => {
-  const apiEndpoint = getApiEndpoint()
-  const client = await getClient(apiEndpoint)
+  const client = await getClient(t.context.apiEndpoint)
+
+  // Get metrics before upload
+  const beforeOperationW3Metrics = await getW3Metrics(t)
+  const beforeW3AccumulatedSize = beforeOperationW3Metrics.find(row => row.name === W3_METRICS_NAMES.STORE_ADD_ACCUM_SIZE)
```

Comment:
Please can we have a separate `test('POST / updates metrics')` for example?

Reply:
I wanted to do what you suggested and should have documented decision in PR description!

This is the integration tests and therefore we write real CAR files (a new one per PR/commit) that will live in S3/R2. Adding more individual tests will lead into more and more files that we will need to pay for storage + egress just for testing. 

Probably I can suggest renaming this file to `integration.test.js` and make test names more appropriate to include everything. Thoughts?

Reply:
Ohhh ok, yeah renaming will be fine!